ethereum/erc-6551/scripts/createAccount.js

```diff
@@ -0,0 +1,46 @@
+const hre = require('hardhat');
+const { ethers } = require('ethers');
+require('dotenv').config();
+
+const provider = new ethers.JsonRpcProvider(process.env.RPC_URL)
+const signer = new ethers.Wallet(process.env.PRIVATE_KEY, provider)
+
+const registryAddress = '0x02101dfB77FDE026414827Fdc604ddAF224F0921';
+const implementationAddress = '0x2d25602551487c3f3354dd80d76d54383a243358';
+const tokenId = 0;
+const salt = 0;
+let nftContract;
+
+const getTokenBoundAccount = async () => {
+  const RegistryContract = await hre.ethers.getContractAt('ERC6551Registry', registryAddress, signer);
```

Comment:
Nit, but since you call `await hre.ethers.getContractAt('ERC6551Registry', registryAddress, signer);` in main, i'd just pass it's value instead of calling it again.
```suggestion
const getTokenBoundAccount = async (RegistryContract) => {
```